### PR TITLE
Add tags data file

### PR DESCRIPTION
**Pull request by nahteb**
- Add a data file for tags to reduce duplication 
- I've added a description to the json schema as we have it in our [data structure](https://github.com/alphagov/data-standards-authority/blob/main/adr/0001-standards-catalogue-data-structure.md), but haven't added any descriptions - I've used what was in the topics, keywords and classifications as the tags and I think we can change those / add descriptions in a separate PR

### CODE REVIEW

File: standards-catalogue/Rakefile [hunk 1]
```diff
@@ -68,9 +68,19 @@ namespace :schema do
...
       raise unless errors.all?
     end
+
+    desc "Validate tags are schema compliant"
+    task :tags do
+      schema = schema("tag")
+      tags = YAML.load_file("data/tags.yml")
+      errors = tags.map do |_, tag|
+        is_valid_schema(schema, tag)
+      end
```
Comment (jamietanna) [inline standards-catalogue/Rakefile:78]: :bell: ding ding ding! We've hit the third time we've duplicated htis code, we should look at this being a common method

**Review by jamietanna — COMMENTED** [review]
Couple of comments.

Also I'd recommend 6ed58af5924b091849360240c8d3fad8907b5d7e being squashed into 8f5930d43cb35e5de9f7e17cf47c6d8f87848fc7 maybe with a title along the lines of "replace classification with tags" - this allows us to make sure that the commit itself is atomic - we wouldn't want the empty Classification column to stay there

File: standards-catalogue/Rakefile (hunk 1 above)
Comment (nahteb) [reply standards-catalogue/Rakefile]: done in https://github.com/alphagov/data-standards-authority/pull/127/commits/092e3f44c856b987a821d7c39c570dfc9afcde34

**Review by jamietanna — APPROVED** [review]
Awesome :raised_hands:

**Pull request merged**